### DISCUSSION TITLE
v0.2.2-preview.0

### DISCUSSION THREAD

**Discussion by cymian** [cymian/mouseless#78] [Announcements]
This is a preview release for v0.2.2.

So far, it contains fixes for:
#39 (Non-english layouts/characters not working)
#30 (Some chars on alternate english layouts (e.g. dvorak) not working)
#52 (minor: updated some menu items' text)

Testing it out and providing feedback before Friday is definitely appreciated -- a big thank you to anyone who does!! 🙏

<hr /><em>This discussion was created from the release <a href='https://github.com/croian/mouseless-issues/releases/tag/'>v0.2.2-preview.0</a>.</em>

**Comment by cymian** [comment]
closing this in favor of #79 (issue instead of discussion) -- still trying to figure out the best way to organize this place 🙃

**Comment by twiggshub** [comment]
Happy to, but unable to test this, replaced the .dmg in my applications folder. Program won't start up. 

Tried replacing the preview .app with the old 0.2.1 release did not work. (I know mac has its quirks with having to approve in security settings, but it doesn't seem to be the case this time) 

Any ideas on what this could be due to?
